src/js/sw.js

```diff
@@ -111,10 +109,20 @@ const handleInternalMessage = async (message) => {
 
     let idpOrWebID = activeIdentity.idp;
     if (activeIdentity.webID) {
-      const idps = await getIDPsFromWebID(activeIdentity.webID);
-      // In the end we set the IDP anyway. User gets redirected to IDP. If it supports more than one WebID, the user can confirm or select the correct one there.
-      if (idps.length) {
-        idpOrWebID = idps[0];
+      try {
+        const idps = await getIDPsFromWebID(activeIdentity.webID);
+        // In the end we set the IDP anyway. User gets redirected to IDP. If it supports more than one WebID, the user can confirm or select the correct one there.
```

Comment:
I don't understand what this comment is clarifying.

Reply:
I'm not sure I understand either. I think the comment should state that if multiple IDPs are available and that by default the first one will be selected.
Agree?

Reply:
If we neither of us understand it, remove it then.